src/GeneralStateTestsFiller/stBadOpcode/validInvalidOpcodeFiller.yml

```diff
@@ -0,0 +1,6008 @@
+
+validInvalidOpcode:
+
+# Do not manually edit this file. It is generated by
+# validInvalidOpcodeGen.js
+
+  _info:
+    comment: |
+      Ori Pomerantz qbzzt1@gmail.com
+      Implements: OPCOD0012, OPCOD0030, OPCOD0073, OPCOD0092, OPCOD0165
+                  OPCOD0246, OPCOD0252, OPCOD0254
+
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: 0x20000
+    currentGasLimit: 0xFF112233445566
+    currentNumber: 1
+    currentTimestamp: 1000
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+    currentBaseFee: 10
+
+
+  pre:
+
+    # Contract called by the test code to have return data
+    0000000000000000000000000000000000ca11ed:
+      balance: 0
+      nonce: 1
+      storage: {}
+      code: |
+          :yul {
+               mstore(0, 0xDEAD60A7)
+               return(0x1C, 4)
+          }
+
+
+    # Contract called by the transaction
+    cccccccccccccccccccccccccccccccccccccccc:
+      balance: 0
+      code: |
+          :yul {
+              let opcodeTestBase := 0x7E57C0DE00000000000000000000000000000000
+
+              // For every possible opcode
+              for { let op := 0x00 } lt(op,0x100) { op := add(op,1) } {
+                 pop(call(60000, add(opcodeTestBase, op), 0, 0,0, 0,0))
+                 sstore(op, returndatasize())
+              }     // for loop
+          }
+      nonce: 1
+      storage: {}
+
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 1_000_000_000_000
+      code: '0x'
+      nonce: 1
+      storage: {}
+
+
+
+    0x7E57C0DE00000000000000000000000000000000:
+       balance: 0
+       nonce: 1
+       storage: {}
+
+       code: |
+         :yul {
+             // A call to have return data
+             pop(call(gas(), 0xca11ed, 0, 0,0, 0,0))
+
+
+             // Run the opcode, assume 20 parameters are enough
+             verbatim_20i_0o(hex"005b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b5b", 1, 1, 3, 4, 5, 6, 7, 8, 9, 10,
+                                                  11,12,13,14,15,16,17,18,19,20)
+
+             // If we get here, return one byte. That would be the sign of success
+             return(0,1)
+         }
+
+
+
+
+    0x7E57C0DE00000000000000000000000000000001:
```

Comment:
is there a reason each time you call a different address? can't the opcode to call be in input argument?